cfme/tests/cloud/test_providers.py

```diff
@@ -108,8 +108,8 @@ def test_type_required_validation(request, soft_assert):
             prov.create()
     else:
         pytest.sel.force_navigate("clouds_provider_new")
-        fill(properties_form.name_text, "foo")
-        soft_assert("ng-invalid-required" in properties_form.type_select.classes)
+        fill(prop_region.properties_form.name_text, "foo")
```

Comment:
Shouldn't this (and the other instances of `prop_region.properties_form`) be just `prov.properties_form`?